### ISSUE TITLE
Isar inspector not working

### ISSUE THREAD

**Issue by ProgUser145** [isar/isar#147]
**Steps to Reproduce**
Hi, thank you for the stable release of Isar. I am having some problems with the most recent version. I just tried to use the sample from the "quotes" app in the docs. When I copy the uri from the terminal that is printed from connecting to the instance into the isar inspector it doesn't do anything when clicking continue. Before with version 0.4.0 it used to redirect me to the collections, but now it seems stuck at the very first step. Is there some additional configuration needed? Or is the isar inspector still a work in progress? Thank you for the help.

**Version**
 - Most recent one (stable)
 - Mac Os

**Comment by leisim** [comment]
That's weird. Do you use the latest version of the inspector?

**Comment by ProgUser145** [comment]
Yes exactly. Does it work normally for you? Then it must be on my side

**Comment by leisim** [comment]
Yes, it works for me but there could still be a bug. I didn't have time to write tests for the inspector yet. It would be super helpful if you could run the inspector from the source and see if you get any errors...

**Comment by ProgUser145** [comment]
I will try that, thank you

**Comment by zawadZwd** [comment]
> Yes, it works for me but there could still be a bug. I didn't have time to write tests for the inspector yet. It would be super helpful if you could run the inspector from the source and see if you get any errors...

Let me help you 😉
I also have problems with inspector, so I downloaded the source code, launched it and here what it goes after I tried to connect to my Flutter app db:

```dart

flutter: Connecting…
flutter: isolates/2620846947896307
flutter: ERROR: Unable to connect to VMService
flutter: type 'Null' is not a subtype of type 'String' in type cast
flutter: #0      _$_$_LinkFromJson package:isar_inspector/schema.g.dart:44
#1      new _$_Link.fromJson package:isar_inspector/schema.freezed.dart:471 
#2      _$LinkFromJson package:isar_inspector/schema.freezed.dart:365
#3      new Link.fromJson package:isar_inspector/schema.dart:36
#4      _$_$_CollectionFromJson.<anonymous closure> package:isar_inspector/schema.g.dart:16
#5      MappedListIterable.elementAt (dart:_internal/iterable.dart:413:31)
#6      ListIterator.moveNext (dart:_internal/iterable.dart:342:26)
#7      new _GrowableList._ofEfficientLengthIterable (dart:core-patch/growable_array.dart:188:27)
#8      new _GrowableList.of (dart:core-patch/growable_array.dart:150:28)
#9      new List.of (dart:core-patch/array_patch.dart:51:28)
#10     ListIterable.toList (dart:_internal/iterable.dart:213:44)
#11     _$_$_CollectionFromJson package:isar_inspector/schema.g.dart:17
#12     new _$_Collection.fromJson package:isar_inspector/schema.freezed.dart:138
#13     _$CollectionFromJson package:isar_inspector/schema.freezed.dart:16
#14     new Collection.fromJson package:isar_inspector/schema.dart:15
#15     _ConnectPageState._connect.<anonymous closure> package:isar_inspector/connect.dart:29
#16     MappedListIterable.elementAt (dart:_internal/iterable.dart:413:31)
#17     ListIterator.moveNext (dart:_internal/iterable.dart:342:26)
#18     new _GrowableList._ofEfficientLengthIterable (dart:core-patch/growable_array.dart:188:27)
#19     new _GrowableList.of (dart:core-patch/growable_array.dart:150:28)
#20     new List.of (dart:core-patch/array_patch.dart:51:28)
#21     ListIterable.toList (dart:_internal/iterable.dart:213:44)
#22     _ConnectPageState._connect
package:isar_inspector/connect.dart:30
<asynchronous suspension>
```

**Comment by zawadZwd** [comment]
Well it is interesting. When I removed from the flutter app schema named 'PinnedContactIsar', the inspector works perfectly fine.

**Comment by leisim** [comment]
Thanks for your investigation! Fixed by #160
